src/number-formatting/short-format.js

```diff
@@ -18,26 +18,29 @@ import formatNumber from './format';
  * @returns {string} Returns 0 when dates are equal. -1 when date1 less than date2. 1 when date1 greater than date2.
  */
 function shortFormat(num, options) {
+    const [digits, decimals] = String(num).split('.');
+    const decimalSize = decimals ? decimals.length : 0;
+    const number = Number(`${digits}e${decimalSize}`);

```

Comment:
Why not Number(digits)
What is this doing? 

105.232 => 105e3 ? == 105000 which it then compares against boundaries?